src/main/java/com/dataloom/edm/EdmApi.java

```diff
@@ -183,6 +184,11 @@ Void removePropertyTypeFromEntityType(
     Schema getSchemaContents(
             @Path( NAMESPACE ) String namespace,
             @Path( NAME ) String name );
+    
+    @GET( SCHEMA_BASE_PATH + NAMESPACE_PATH + NAME_PATH + YAML_PATH )
+    String getSchemaContentsAsYaml(
```

Comment:
query param just like we do for downloading data. We already have an enum somewhere. @hochung can point you at it.